mla/test_statistics.py

```diff
@@ -243,45 +246,40 @@ def i3_test_statistic(params: np.ndarray, prepro: I3Preprocessing,
         params, dtype=prepro.params.dtype, copy=True)
     sob = prepro.sob_spatial * _sob_time(temp_params, prepro) * np.exp(
         [spline(temp_params['gamma']) for spline in prepro.splines])
+
+    if 'ns' in temp_params.dtype.fields:
```

Comment:
I think this works, but use `names` instead of `fields` to be sure and to match the rest of the code